node/sdk/pb/base.proto

```diff
@@ -0,0 +1,81 @@
+// Base types for basic SDK functionality
+
+syntax = "proto3";
+
+package pb;
+
+message CheckAccountRequest {
+    string name = 1;
+}
+
+message CheckAccountReply {
+    string name = 1;
+    string chain = 2;
+    bytes accountKey = 3;
+    // TODO: Should use Coin message
+    Balance balance = 4;
+}
+
+enum Currency {
```

Comment:
Can you make this come from the /node/data/coin.go? so when we add new currency, we don't need to change both place?